contracts/stellar-interchain-token-service/src/tests/deploy_remote_canonical_token.rs

```diff
@@ -162,7 +162,7 @@ fn deploy_remote_canonical_token_succeeds_without_name_truncation() {
 }
 
 #[test]
-#[should_panic(expected = "HostError: Error(Storage, MissingValue)")]
+#[should_panic(expected = "HostError: Error(Contract, #16)")] // InvalidTokenAddress
 fn deploy_remote_canonical_token_fail_no_actual_token() {
```

Comment:
Doesn't `register_canonical_token` fail now and not deploy_remote_canonical_token? Refactor the test accordingly. Also, use `assert_contract_err` instead of testing for panic